signature-aggregator/api/api.go

```diff
@@ -150,5 +166,8 @@ func signatureAggregationAPIHandler(logger logging.Logger, aggregator *aggregato
 		if err != nil {
 			logger.Error("Error writing response", zap.Error(err))
 		}
+		metrics.AggregateSignaturesLatencyMS.Set(
+			float64(time.Since(startTime) / time.Millisecond),
```

Comment:
```suggestion
			time.Since(startTime).Milliseconds(),
```
May need to cast to `float64` still

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/398/commits/b37ce9cf7908f6c4b7455d8c760287cc691f9420